sdk/java-sdk/src/main/java/com/akkaserverless/javasdk/action/ActionContext.java

```diff
@@ -35,7 +35,7 @@
    *
    * @return The call level metadata.
    */
-  Metadata metadata();
+  Metadata metadata(); // TODO this is not necessary
```

Comment:
It has better docs than the MetadataContext one though :)

Reply:
hehe 
Shall I add in there `FIX: remove and move docs from this method class to MetadataContext`? for the moment?

Reply:
Sure

Reply:
ok